### ISSUE TITLE
ng-class and class bugs

### ISSUE THREAD

**Issue by ruwhan** [angular/angular.js#12897]
This issue is related with _jqplot_,

Following code will make the jqplot appear
```
ng-class="{'success collapse row-details': build.state == 'Complete' || build.state == 'Accepted', 'info collapse row-details': build.state == 'Running', 'danger collapse row-details': build.state == 'Rejected'}"
```

But **not** with this code:
```
ng-class="{'success': build.state == 'Complete' || build.state == 'Accepted', 'info ': build.state == 'Running', 'danger': build.state == 'Rejected'}" class="collapse row-details"
```

As additional note, another element appear normally, just the _jqplot_ chart missing.

It is seems this is an Angular bug.

EDIT: I'm using Angular 1.3.0

**Comment by Narretz** [comment]
Can you give a demo of that via plnkr.co or similar? Also, how exactly does do you integrate the jqplot? Finally, you should always use the latest patch version at least, so 1.3.19, or even better upgrade to 1.4.x. It's not guaranteed it'll fix your issue, but if it's really a bug with angular, we won't create a fix for 1.3.x, as we are currently on 1.5.x and only 1.4.x receives fixes.

**Comment by Narretz** [comment]
I am gonna close this issue due to lack of response. Please feel free to reopen if you can provide more info.
